util/db/dbutil.go

```diff
@@ -145,12 +145,15 @@ func (db *Accessor) Close() {
 // Sends warnings and errors to log.
 func LoggedRetry(fn func() error, log logging.Logger) (err error) {
 	for i := 0; ; i++ {
-		if i > 0 && i%warnTxRetries == 0 {
```

Comment:
nit: reduce amount of branches with something like this:
```
conn, err = db.Handle.Conn(ctx)
for i := 1; dbretry(err); i++ {
  // logging
  conn, err = db.Handle.Conn(ctx)
}
```

Reply:
good idea.